src/js/screen_gfx.js

```diff
@@ -2,7 +2,9 @@
 
 import rgba from "color-rgba";
 
-const fontFamily = '"sans-serif-condensed", "Arial Narrow", sans-serif';
+const naskhFonts =
+  '"Noto Naskh Arabic", KacstNaskh, "Sakkal Majalla", "DecoType Naskh Regular"';
```

Comment:
I picked a few Naskh fonts that are available by default on most operating systems. I'm not 100% sure these are all the best choices, but they look close enough to the style as shown on Saudi shields.

Reply:
Legibility doesn't look good on my system:

![image](https://user-images.githubusercontent.com/3254090/230738849-56d21a3f-05fd-4cb8-b61e-6ff2b3a8b0bd.png)

Why not use the packaged Americana fontstacks that have Arabic?

Reply:
Shields are rendered client-side using fonts available to the browser. We would have to re-engineer shields to render text from a PBF fontstack instead of standard web formats.

Reply:
Oh duh good point.  Don't you need some kind of `<link>` tag to use the Google Noto fonts?

Reply:
Something like
```
<link href='https://fonts.googleapis.com/css?family=Noto Sans' rel='stylesheet'>
```

Reply:
Not if it's already installed on the system, as with Android. But downloading and displaying Noto on all systems is an option.

Reply:
Is there a reason we wouldn't? I assume it's just a one time download into browser cache.

Reply:
I suppose not. Though we might as well include Noto Sans Armenian as well, if consistency is what we're going for.

Reply:
I guess we'll need whatever scripts would get rendered into shields.